lib/simulator-xcode-6.js

```diff
@@ -478,6 +478,13 @@ class SimulatorXcode6 extends EventEmitter {
     await B.all(deletePromises);
   }
 
+  /**
+   * Remove the apps-related keychain.
+   */
+  async cleanAppsKeychain () {
+    await fs.rimraf(path.resolve(this.keychainPath, 'keychain-2-debug.db'));
```

Comment:
is this name always the same? perhaps it would be better to use glob expression to match it

Reply:
[This SO issue](https://stackoverflow.com/questions/41862997/ios-simulator-view-content-of-keychain) and [this other SO issue](https://stackoverflow.com/questions/1636506/how-to-see-the-keychain-for-iphone-simulator) seem to indicate it. The second one points out in the comments that this path is only valid starting with the iOS 8 SDK (that comes with XCode 6). Removing this file certainly worked for us.

Would you prefer `*.db` instead of `keychain-2-debug.db`?

Reply:
`keychain*.db` looks better